### PR TITLE
Create day 4 slides

### PR DESCRIPTION
**Pull request by brookskindle**
You can also view the slides locally if you set up a local server and point your browser to localhost:8000.

More complete instructions can be found at https://github.com/Ada-Developers-Academy/jump-start-live/tree/master/docs#i-made-changes-how-can-i-view-the-slides-locally

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -45,6 +55,7 @@
     </script>
     <script type="text/javascript">
       const options = {
+        highlightLines: true
```
Comment (brookskindle) [inline docs/index.html]: The contents in this file are identical to those in #31, but it hasn't been approved and merged yet.

**Review by beccaelenzil — APPROVED** [review]

File: docs/slides/day4.md [hunk 2]
```diff
@@ -0,0 +1,524 @@
...
+
+total = 0
+input = gets.chomp.to_i
+confirmedInput = gets.chomp.to_i
+
+while true # infinite loop, quit on break
+  if input == confirmedInput
+    if input < 0 # time to quit
+      break
+        else # add to total
```
Comment (CheezItMan) [inline docs/slides/day4.md:47]: Should this section be aligned with the `if`?

**Review by CheezItMan — APPROVED** [review]
Thanks for getting on this.  I think the markdown slides will be useful, once I get a handle on all of them.

File: docs/slides/day4.md (hunk 2 above)
Comment (brookskindle) [reply docs/slides/day4.md:47]: Normally yes, but this is on a code review slide, so that's a purposeful mistake.

**Pull request merged**